ui/imports/shared/controls/Input.qml

```diff
@@ -177,9 +177,10 @@ Item {
         id: validationErrorText
         visible: !!validationError
         text: validationError
+        anchors.left: inputField.left
```

Comment:
Just a thought here if it's better to use Layouts instead of anchoring as anchoring are less flexible in general.

Reply:
Yeah but it's a really old component and unrelated to this change